package/src/DataTable/DataTable.js

```diff
@@ -353,12 +363,19 @@ const DataTable = React.forwardRef(function DataTable(props, ref) {
                   return (
                     <TableCell
                       padding={column.id === "selection" ? "checkbox" : "default"}
-                      classes={{
-                        root: classes.tableHead
-                      }}
+                      className={clsx({
+                        [classes.tableHead]: true,
+                        [classes.tableHeadClickable]: !!column?.isSortable
```

Comment:
@hrhosni we're currently not using optional chaining in this project, the babel config and node version we're using doesn't support it and I'm seeing a failing test in CI because of this use. Can you please refactor to not use optional chaining here.